data/json/items/tools.json

```diff
@@ -5786,15 +5790,15 @@
     "description": "This is a sealed glass jar containing sliced fruits.  Use to open and eat to enjoy.",
     "price": 500,
     "material": ["glass", "veggy"],
-    "weight": 600,
+    "weight": 1050,
     "volume": 2,
     "bashing": 4,
     "category" : "food",
     "use_action": {
         "type": "transform",
         "msg": "You open the jar, exposing it to the atmosphere.",
         "target": "apple_canned",
-        "target_charges": 3,
+        "target_charges": 2,
```

Comment:
It takes 3 volume of fruit and is noticeably less nutritious than other canned items. If it loses that 1 extra charge, it should gain it back in nutrition.